vocab_list/models.py

```diff
@@ -73,8 +73,12 @@ def load_tab_delimited(self, fd, entry_model, extra_attrs={}):
         Returns: bulk create query of vocab list entries
         """
         lines = [parse_line(idx, line) for idx, line in enumerate(fd)]
-        existing_headwords = entry_model.objects.filter(vocabulary_list=self).values_list("headword", flat=True)
-        new_headwords = filter(lambda x: x[0] != "" and x[0] not in existing_headwords, lines)
+        # filter out duplicates in the files
+        # how do we know its not a real duplicate vs word duplicate with different definitions? sets!
+        filter_repeats_lines = list(set(lines))
```

Comment:
This is a nice approach to eliminating duplicates. Just a note that `set()`works because `lines` consists of a series of 2-tuples returned by `parse_line()` and tuples are immutable/hashable. If that implementation changes in the future (e.g. the tuple is changed to a mutable list), then this could break. 

Compare:

```python
>>> set([("quam", "as possible as"), ("quam", "how"), ("quam", "how")])
{('quam', 'how'), ('quam', 'as possible as')}
>>> set([["quam", "as possible as"], ["quam", "how"], ["quam", "how"]])
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: unhashable type: 'list'
```

The good thing is we have some tests, so that should be caught if it happens.

Reply:
Ohh i see! Good to know for the future! So data structure that is considered mutable(list, dict) would not work in a set. Ex: list of dicts
```python
set([{"id": 1}, {"id": 1}, {"id": 2}])
```